application/controllers/QuestionAdministrationController.php

```diff
@@ -2973,14 +2973,6 @@ private function getQuestionTypeGroups(array $questionThemes)
                     'questionGroupName' => $questionTheme->group
                 );
             }
-            $imageName = $questionTheme->question_type;
```

Comment:
Why remove this part ?

Reply:
Only used in this function and only if == 'S' so seems redundant to assign a separate variable